requirements/edx/testing.txt

```diff
@@ -981,7 +981,7 @@ openedx-django-require==2.1.0
     # via -r requirements/edx/base.txt
 openedx-django-wiki==2.0.3
     # via -r requirements/edx/base.txt
-openedx-events==8.8.0
+openedx-events==8.9.0
```

Comment:
This was manually updated as the PR is cleaner than running `make upgrade` which will also upgrade non-related dependencies. Also, the latest version is `9.0.0` but this includes breaking changes related to event bus configuration, and edx needs to update their operation before upgrading this requirement.